packages/acceptance/src/pages/ButtonsPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from '@hig/button';
+import Spacer from '@hig/spacer';
```

Comment:
should `Spacer` also be in the `package.json`?

Reply:
Yeah good catch.